src/js/components/agencyV2/overview/BarChart.jsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * BarChart.jsx
+ * Created by Max Kendall 4/13/21
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const BarChart = ({
+    agencyBudgetByYear,
+    selectedFy
+}) => {
+    const renderBars = () => {
+        const highestYear = agencyBudgetByYear.reduce((acc, obj) => {
```

Comment:
This is a pretty cool way of finding the max. I would not have thought to use reduce.

Calling this variable `highestYear` sounds like this is the greatest year instead of the max budget.  Can we be more descriptive of exactly what this is returning? `maxAgencyBudget` or `greatestAgencyBudget` or `greatestBudget`?

Reply:
Yep agreed.